src/commands/jobs/util.ts

```diff
@@ -286,3 +286,24 @@ export const itemOutputFormat = {
 		'transferred_on'
 	]
 };
+
+export const securityMappingOutputFormat = {
+	table: [
+		{
+			header: 'Source',
+			property: 'source.email'
```

Comment:
`email` is not guaranteed to be the property that is present. I would set `property: 'source'` and use `transfer: val => val && (val.email || val.username || val.id)`